drivers/SmartThings/zigbee-switch/src/zigbee-dual-metering-switch/init.lua

```diff
@@ -1,57 +1,46 @@
--- Copyright 2022 SmartThings
---
--- Licensed under the Apache License, Version 2.0 (the "License");
--- you may not use this file except in compliance with the License.
--- You may obtain a copy of the License at
---
---     http://www.apache.org/licenses/LICENSE-2.0
---
--- Unless required by applicable law or agreed to in writing, software
--- distributed under the License is distributed on an "AS IS" BASIS,
--- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
--- See the License for the specific language governing permissions and
--- limitations under the License.
+local capabilities = require "st.capabilities"
+local ZigbeeDriver = require "st.zigbee"
+local defaults = require "st.zigbee.defaults"
+local device_lib = require "st.device"
 
-local ZIGBEE_DUAL_METERING_SWITCH_FINGERPRINT = {
-  {mfr = "Aurora", model = "DoubleSocket50AU"}
-}
-
-local function can_handle_zigbee_dual_metering_switch(opts, driver, device, ...)
```

Comment:
It's still a sub-driver. You've seemingly redefined a base driver as a sub-driver? I think this will cause you a lot of issues.